origocli/commands/datasets/boilerplate/boilerplate.py

```diff
@@ -0,0 +1,171 @@
+import os
+import re
+import json
+import shutil
+from questionary import prompt
+
+from origocli.command import BaseCommand, BASE_COMMAND_OPTIONS
+from origocli.date import date_now, DATE_METADATA_EDITION_FORMAT
+
+from .config import available_pipelines, boilerplate_questions
+
+
+class DatasetsBoilerplateCommand(BaseCommand):
+    __doc__ = f"""
+usage:
+   origo datasets boilerplate <name> [--file=<file> --prompt=<prompt> --pipeline=<pipeline> options]
+
+Examples:
+  origo datasets boilerplate oslo-traffic-data
+  origo datasets boilerplate oslo-traffic-data --file=/tmp/initial-file.csv
+  origo datasets boilerplate oslo-traffic-data --pipeline=data-copy --prompt=no
+  origo datasets boilerplate geodata-from-my-iot-devices
+
+Options:{BASE_COMMAND_OPTIONS}
+  --file=<file>         # Initial file to upload
+  --prompt=<prompt> # Use input prompt to collect data, default "yes"
+  --pipeline=<pipeline>  # Required when --prompt=no
```

Comment:
Hva om man angir denne uten `prompt=no`? Da blir man spurt i wizard'en om pipeline på nytt?

Reply:
for no så blir du det, ja, med 2pipelines tilgjengleg så ser eg ikkje store problemet her heilt enda